src/Node/ClassPropertiesNode.php

```diff
@@ -96,7 +96,7 @@ public function getUninitializedProperties(
 			if ($property->isStatic()) {
 				continue;
 			}
-			if ($property->getNativeType() === null) {
+			if ($property->getNativeType() === null && !$property->isReadonlyByPhpDoc()) {
 				continue;
 			}
```

Comment:
this is weird, isn't it? it is currently needed to make `UninitializedPropertyRule` work. But this is very native readonly specific IMO, maybe we should not support phpdoc readonly there?

Reply:
I decided to drop phpdoc readonly support in `MissingReadOnlyPropertyAssignRule` again since it seems to handle specific native readonly behaviour.